packages/now-cli/test/dev/integration.js

```diff
@@ -651,8 +670,9 @@ test(
 
     validateResponseHeaders(t, response);
 
-    const body = await response.text();
-    t.regex(body, /Hugo on ZEIT Now/gm);
+    // const body = await response.text();
+    // t.regex(body, /Hugo on ZEIT Now/gm);
```

Comment:
Any reason you're not checking the body text anymore?

Reply:
same as above